Project.toml

```diff
@@ -1,6 +1,12 @@
 name = "ClimaComms"
 uuid = "3a4d1b5c-c61d-41fd-a00a-5873ba7a1b0d"
-authors = ["CliMA Contributors <clima-software@caltech.edu>"]
+authors = [
+    "Kiran Pamnany <clima-software@caltech.edu>",
```

Comment:
@kpamnany would you like your personal email here (and on other repos)? or happy to leave the clima one?